service/cert-reloader.go

```diff
@@ -126,19 +126,20 @@ func (w *CertReloader) loadLocalCertAndKey() error {
 	if err != nil {
 		return errors.Wrap(err, fmt.Sprintf("unable to load cert from %s,%s", w.certPath, w.keyPath))
 	}
-	certPEM, err := os.ReadFile(w.certPath)
-	if err != nil {
-		return errors.Wrap(err, fmt.Sprintf("unable to load cert from %s", w.certPath))
-	}
-	keyPEM, err := os.ReadFile(w.keyPath)
-	if err != nil {
-		return errors.Wrap(err, fmt.Sprintf("unable to load key from %s", w.keyPath))
-	}
+	// TODO: I have this feeling that the following code does not care of reading files properly:
```

Comment:
```suggestion
	// TODO: unused currently, refactor later in garm v4
```
Can we change the comment to some next action?

Reply:
Handled in https://github.com/AthenZ/garm/pull/52/commits/7e781897b592a1622ba950db50e8f9b17402ffe0